### ISSUE TITLE
Docker image cann't run as non-root

### ISSUE THREAD

**Issue by caskolkm** [AxonIQ/axon-server-dockerfile#6]
In order to store our axon data to a NFS, we need to run axon-server as non-root user. After change te user in kubernetes or in docker with `--user`, axon will not run properly.

```
/opt/axonserver/startup.sh: line 20: /opt/axonserver/axonserver.properties: Permission denied
/opt/axonserver/startup.sh: line 48: /opt/axonserver/axonserver.properties: Permission denied
/opt/axonserver/startup.sh: line 49: /opt/axonserver/axonserver.properties: Permission denied
```

**Comment by caskolkm** [comment]
It is already discussed in: https://groups.google.com/forum/#!searchin/axonframework/axon-server%7Csort:date/axonframework/y6f7lHEqKLU/nCLCqFODAwAJ

**Comment by bert-laverman** [comment]
The discussion you refer to just asks for the files in this repo.

If you want a configurable username, you'd have to change the startup.sh script to change to that user, but also make sure all initial files have the correct rights or ownership. This would mean `chown`ing the jars, the properties file, and the startup script.

Do you want to file a PR to that effect?

**Comment by caskolkm** [comment]
@bert-laverman true, but the problem is still the same. The docker-image of axon-server depends on the root-user.

First, this repo on itselfs should be fixed, because it's doing copies of file that doesn't exists in this repo, for example: jar-files. This makes it harder to correctly test.

It's pretty complex to fix this. There are a lot, a lot of ways to fix this problem.

**Comment by bert-laverman** [comment]
Actually, only the JAR files aren't in here, as they should come from the Axon Server build. That will stop being a problem when the source code for Axon Server SE is put online.
